### PR TITLE
Reintroducing gauge increment and decrement.

### LINKED ISSUE #11: With version 2.0.0, gauge increment is not availible
This means that the client is required to keep track of a gauge over long periods of time, potentially across multiple processes. I was under the impression that incrementing a gauge is typical behavior of the statsd protocol. Is it something we can no longer support?

### PR DESCRIPTION
**Pull request by pulltab**
Back by popular demand.  Gauge increments/decrements are emitted according to
etsy/statsd standard.

Closes #11

### CODE REVIEW

**Comment by pulltab** [comment]
@blt @tsantero @daveauerbach r+?

File: pystatsd/pystatsd.py [hunk 1]
```diff
@@ -177,15 +181,19 @@ def gauge(self, stat, value, sample_rate=1):
...
         Updates one or more stats counters by arbitrary amounts
         >>> statsd_client.update_stats('some.int',10)
         """
         if not isinstance(stats, list):
             stats = [stats]
 
-        data = dict((stat, "%s|c" % delta) for stat in stats)
+        if gauges:
+            prefix = '+' if delta and delta >= 0 else ''
+            data = dict((stat, "%s%s|g" % (prefix,delta)) for stat in stats)
```
Comment (pulltab) [inline pystatsd/pystatsd.py]: @blt @tsantero Can you sign off on the wire protocol?

File: pystatsd/test_pystastd.py [hunk 2]
```diff
@@ -27,6 +27,16 @@ def test_counter_decrement(self):
...
         pystatsd.decrement(stat="bar.baz", rate=0.5)
 
+    def test_gauge_increment(self):
+        pystatsd.increment(stat="foo.bar", gauge=True)
+        pystatsd.increment(stat="foo.bar", delta=10, gauge=True)
+        pystatsd.increment(stat="foo.bar", rate=0.5, gauge=True)
+
+    def test_gauge_decrement(self):
+        pystatsd.decrement(stat="bar.baz", gauge=True)
+	pystatsd.decrement(stat="foo.bar", delta=130, gauge=True)
```
Comment (tsantero) [inline pystatsd/test_pystastd.py]: whitespace issue

Comment (pulltab) [inline pystatsd/test_pystastd.py]: Fixed.

File: pystatsd/pystatsd.py (hunk 1 above)
Comment (blt) [inline pystatsd/pystatsd.py]: We'll need to implement this on cernan's side for it to take effect but I'm onboard with the wire-level representation.

**Pull request merged**